run.py

```
@@ -0,0 +1,68 @@
+
+def setup(verbose_logging=False, script="implementation.py"):
+    network_setup(config, verbose_logging)
+    logging.info("Network is up.\n")
+    setup_db()
+    logging.info("DB is up.\n")
+    args = " ".join(sys.argv[1:])
+    addon = None
+    if "-w" not in args:
+        flow_name = f"autosaved_{int(time.time())}.flow"
+        logging.warning(f"Logging flow automatically to: {flow_name}")
+        addon = rf" -w {flow_name}"
```

Ruff

Local variable `addon` is assigned to but never used

Remove assignment to unused variable `addon`

(F841)